appdatabase/migrations/sql/1687249080_add_position_accounts.up..sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS keypairs_accounts_new (
```

Comment:
Isn't `ALTER TABLE keypairs_accounts ADD COLUMN position INT;` working?

Reply:
It will work but it won't update previous position record, see the 
`ROW_NUMBER () OVER (ORDER BY created_at) as position`

Reply:
Let me try to do the insert only

Reply:
Yeah that's what Stefan is saying and what I am doing